contrib/dist/linux/buildrpm.sh

```diff
@@ -7,12 +7,55 @@
 #
 
 #
+# External script parameters
+# The folowing parameters could be used to affect script behaviour.
+# Please, do NOT set the same settings with parameters and config vars.
+#
+# -b
+#    If you specify this option, all-in-one binary RPM would be created
+#    instead of source RPM (SRPM). This option is necessary for other
+#    parameters to work, as they won't be influence SRPM package.
+#
+# -n <name>
+#    This option will change the name of the produced RPM to the <name>.
+#    It is useful to use with "-o" and "-m" options if you want to have multiple
+#    OpenMPI versions installed simultaneously in the same enviroment.
+#
+# -o
+#    With this option the install path of the binary RPM will be changed
+#    to /opt/<name>/<version>
+#
+# -m
+#    RPM would also install modulefiles to location specified in RPM spec file.
+#
+# -d
+#    Script will also build debuginfo RPM.
+#
+# -u
+#    Build RPM with usNIC support. It will include all the recommended compile
+#    options for usNIC as well.
```

Comment:
As much as I appreciate the Cisco-specific option here 😄 , it seems like we should either do this for other vendors, or we shouldn't do it for any of them.  I'd personally be ok removing this Cisco-specific feature and relegating that kind of functionality to build scripts here inside Cisco.

Anyone else have an opinion on this?